patches/master_patch.patch

```diff
@@ -1600,42 +1569,23 @@ index 85ece52a001652a3414a47189b2711f215a66290..960abaeab72bbf45b8b460fbbaf1cdb4
  
    if ([theEvent type] == NSFlagsChanged) {
      // Ignore NSFlagsChanged events from the NumLock and Fn keys as
-@@ -2449,14 +2468,18 @@ - (void)showLookUpDictionaryOverlayFromRange:(NSRange)range
-   // do so. For this reason, for now, we accept this non-ideal way of fixing the
-   // point offset manually from the view bounds. This should be revisited when
-   // fixing issues in TextInputClientMac (https://crbug.com/643233).
-+#ifdef MUON_CHROMIUM_BUILD
-   gfx::Rect root_box = renderWidgetHostView_->GetViewBounds();
-   gfx::Rect view_box = widgetHost->GetView()->GetViewBounds();
-+#endif  // MUON_CHROMIUM_BUILD
- 
-   TextInputClientMac::GetInstance()->GetStringFromRange(
-       widgetHost, range, ^(NSAttributedString* string, NSPoint baselinePoint) {
-+#ifdef MUON_CHROMIUM_BUILD
-         baselinePoint.x += view_box.origin().x() - root_box.origin().x();
-         baselinePoint.y +=
-             root_box.bottom_left().y() - view_box.bottom_left().y();
-+#endif  // MUON_CHROMIUM_BUILD
-         [self showLookUpDictionaryOverlayInternal:string
-                                     baselinePoint:baselinePoint
-                                        targetView:targetView];
-@@ -2835,6 +2858,11 @@ - (RenderWidgetHostViewMac*)renderWidgetHostViewMac {
+@@ -2838,6 +2855,11 @@ void RenderWidgetHostViewMac::OnDisplayMetricsChanged(
```

Comment:
although to be honest I'd like to get rid of these patches, just want to make sure we don't break anything